JoinColony/colonyCDapp
# test: create colony flow

## Description

- Added e2e tests for the Create Colony flow
- Set up playwright eslint plugin and dotenv 

## Testing

**Prerequisites**: The test script simulates user interactions with the local instance of the web app, which means that the BE is expected to be set up and running. For now, backend services should be started manually and the initial test data should be seeded . Therefore, ensure that the development environment is properly set up by following the instructions in the README file

- Setup and start your local dev environment  (all BE services should be up and running)  `npm run dev` 
- Seed the data, if it's not already done before `node scripts/create-data.js` 
- If it's the first time you run e2e tests on your machine - run `npm e2e:install` 

- run `npm run e2e` or `npm run e2e:ui`
- Ensure all tests pass

## Diffs

**New stuff** ✨

- playwright folder is added to the lint script
- added loading of the .env file into playwright environment
